### PR TITLE
Feature v0.15.1 apidocs

### PR DESCRIPTION
**Pull request by cameronts**

### CODE REVIEW

**Comment by afeiszli** [comment]
@cameronts merge conflicts

**Comment by cameronts** [comment]
> @cameronts merge conflicts

resolved

**Comment by cameronts** [comment]
> > @cameronts merge conflicts
> 
> resolved

File: auth/auth.go [hunk 1]
```diff
@@ -96,7 +97,14 @@ func HandleAuthCallback(w http.ResponseWriter, r *http.Request) {
 	functions[handle_callback].(func(http.ResponseWriter, *http.Request))(w, r)
 }
 
-// HandleAuthLogin - handles oauth login
```
Comment (0xdcarns) [inline auth/auth.go:99]: Is it possible to leave public function comments?

Comment (cameronts) [reply auth/auth.go:99]: you mean like this?  If so, seems to work..  I can put the Swagger comments before the regular function comments.

// swagger:route GET /api/oauth/login nodes HandleAuthLogin
//
// Handles OAuth login
//
//		Schemes: https
//
// 		Security:
//   		oauth
// HandleAuthLogin - handles oauth login

**Review by afeiszli — APPROVED** [review]

**Pull request merged**